src/NumericalAlgorithms/SphericalHarmonics/Tags.hpp

```diff
@@ -358,3 +358,24 @@ using compute_items_tags =
                D2xRadiusCompute<Frame>, LaplacianRadiusCompute<Frame>,
                NormalOneFormCompute<Frame>, TangentsCompute<Frame>>;
 }  // namespace StrahlkorperTags
+
+/// Tags related to symmetric trace-free tensors
+namespace STF::Tags {
```

Comment:
While we don't have any strict policies on namespaces, we prefer either `Stf` or `stf`